app/dockerdwrapper.c

```diff
@@ -698,12 +709,10 @@ main(void)
   }
 
   while (application_exit_code == EX_KEEP_RUNNING) {
-    if (dockerd_process_pid == -1 &&
-        !read_settings_and_start_dockerd(&app_state)) {
-      quit_program(EX_SOFTWARE);
-    } else {
-      g_main_loop_run(loop);
-    }
+    if (dockerd_process_pid == -1)
+      read_settings_and_start_dockerd(&app_state);
+
+    g_main_loop_run(loop);
```

Comment:
I'm still not sure about how we should handle if read_settings_and_start_dockerd fails - should we not stop the application in at least some cases, e.g. wrong file permissions

Reply:
We will report errors in a status parameter in the future. Then the application should keep running as long as the user can recover from the error.